### ISSUE TITLE
Hey!

### ISSUE THREAD

**Issue by sklrsn** [clickot/ungoogled-chromium-build#3]
Hey!,
    I couldn't find a way to reach you out. I just created this issue to say Thanks for maintaining this repo. 

Can we get these diffs to the official repo ?. I think, the official repo is rather unmaintained or almost dead.

**Comment by clickot** [comment]
Hi @sklrsn , thanks for contacting me. The ungoogled-chromium-portable and my repo do not have so much in common any more :-). Of course we could add my code to this repo, but maybe it should be discussed this with @networkException, @Ahrotahn or @Eloston.

**Comment by networkException** [comment]
Upstreaming is always welcome

**Comment by clickot** [comment]
@networkException thanks! So the proper way of doing this would be fork ungoogled-chromium-portable,  push my code to it and create a PR though my code and the repo code has nearly nothing in common any more?

**Comment by Eloston** [comment]
@clickot Yeah feel free to push what you feel is appropriate. Just assign me the PR and I'll take a look. We can grant you write/admin access to the repo too (as long as the team is fine with it)

As @sklrsn mentioned, ungoogled-chromium-portablelinux is pretty much dead. I only occasionally update it whenever I want to update https://github.com/ungoogled-software/ungoogled-chromium-debian/tree/unportable (because that branch is a fork of portablelinux that I mantain)
